app/main/routes_frontend.py

```diff
@@ -376,7 +376,7 @@ def delete_pico_recipe():
 def available_networks():
     # TODO: properly handle failures by hiding settings in /setup or showing error
 
-    wifi_list = subprocess.check_output('./scripts/pi/wifi_scan.sh', shell=True)
+    wifi_list = subprocess.check_output('./scripts/pi/wifi_scan.sh | grep 2.4', shell=True)
```

Comment:
filter to just show those scanned networks on 2.4ghz... cause all 5.0ghz are effectively useless for this project